test/custom-renderers/send-html.js

```diff
@@ -0,0 +1,12 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
```

Comment:
The copyright should say 2018 in all newly added files.
Please move custom renderers to a different location - it's our convention to put test fixtures in `test/fixtures` directory:

```
test/fixtures/custom-renderers
```

Please use file names that are different from files containing the built-in renderers, for example:

```
custom-send-html.js
send-html.custom.js
```

Reply:
👍